### ISSUE TITLE
Integrate JSLint into plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#12]
```
Adapt https://github.com/happygiraffe/jslint4java-eclipse

```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 2 Feb 2011 at 5:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 2 Feb 2011 at 5:53
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2 release
```

Original comment by `paul.beu...@gmail.com` on 23 Feb 2011 at 4:24
- Changed state: **Fixed**
